collectors/ebpf.plugin/ebpf.c

```diff
@@ -1175,26 +747,28 @@ int main(int argc, char **argv)
     int i;
     int end = NETDATA_EBPF_PROCESS_THREADS;
 
-    void * (*function_pointer[])(void *) = {process_publisher, process_collector, process_log };
+    void * (*function_pointer[])(void *) = { ebpf_process_thread, ebpf_socket_thread };
```

Comment:
you already have this in `ebpf_module_t ebpf_modules[]` as `.start_routine` ?

Reply:
```C
ebpf_module_t ebpf_modules[] = {
    { .thread_name = "process", .enabled = 0, .start_routine = ebpf_process_thread, .update_time = 1,
      .global_charts = 1, .apps_charts = 1, .mode = MODE_ENTRY, .probes = process_probes },
    { .thread_name = "socket", .enabled = 0, .start_routine = ebpf_socket_thread, .update_time = 1,
      .global_charts = 1, .apps_charts = 1, .mode = MODE_ENTRY, .probes = socket_probes },
    { .thread_name = NULL, .enabled = 0, .start_routine = NULL, .update_time = 1,
      .global_charts = 0, .apps_charts = 1, .mode = MODE_ENTRY, .probes = NULL },
};
```

Reply:
Good catch, thank you!